lib/ansible/module_utils/facts/hardware/linux.py

```diff
@@ -483,7 +471,28 @@ def get_mount_facts(self):
 
             mount_info.update(mount_statvfs_info)
 
-            mounts.append(mount_info)
+            return mount_info
+
+    @timeout.timeout()
```

Comment:
No longer need timeout, I think because we'll use timeouts from multiprocessing.

Reply:
both, keep a global one for rest of function

Reply:
but on the other hand, its same length of time .. so it might not make sense anymore, since we do it by mount vs for all mounts